Assets/Scripts/Scriptables/Systems/ShipShieldRepairSystem.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Celeritas.Game;
+using Celeritas.Game.Entities;
+using Celeritas.Scriptables.Interfaces;
+using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace Celeritas.Scriptables.Systems
+{
+
+	[CreateAssetMenu(fileName = "New Ship Shield Repair Modifier", menuName = "Celeritas/Modifiers/Ship Shield Repair")]
+
```

Comment:
Would be nice to have a summary comment of the entire class here, describing generally what it does. (a bit obvious, I know, but nice to have c: )

Reply:
I've committed the changes and added a summary for the class.
Not sure on what the shape needs to be but we can change it later if required.